docs/reference/resource-limits-fees.mdx

```diff
@@ -13,26 +13,49 @@ Resource limitations and fees only apply to smart contract transactions. Read mo
 
 Soroban introduces smart contracts, marking the most transformative upgrade to the Stellar network to date. To maintain network performance, the ecosystem has decided to upgrade the network in a phased approach where network capacity is increased gradually over time. See [this blog post](https://stellar.org/blog/developers/the-stellar-network-s-phased-rollout-of-smart-contracts-the-road-to-mainnet) for more details and [Discord](https://discord.com/invite/zVYdY3ktTn) for current status.
 
-| Network Setting | Phase 0 | Phase 1 (current) |
-| :-- | :-- | :-- |
-| Soroban Txn per ledger | 1 | 100 |
-| Max CPU Instructions per Txn | 2.5 million (2M instructions for max Wasm) | 100 million (VM instantiation consumes up to 23M for a 64KB Wasm, and 46M for multiple Wasms of 130KB) |
-| Memory limit per Txn | 2 MB | 40 MB |
-| Ledger entry size (including Wasm entries) per Txn | 2 KB | 64 KB |
-| Read/Write Ledger entries per Txn | 3 read; 2 write | 40 read; 25 write |
-| Read/Write bytes per Txn | 3.2 KB read; 3.2 KB write | 130 KB read; 65 KB write |
-| Transaction size | 10 KB | 70 KB |
-| Persistent entry minimal/initial lifetime | 4,096 ledgers (~5.68 hours) | 120 days |
-| Temporary entry minimal/initial lifetime | 16 ledgers | 1 day |
-| Max ledger entry expiration bump | 61 days | 6 months |
-| Events+return value size bytes | 200 B | 8 KB |
+| Network Setting | Phase 0 | Phase 1 (current) | Phase 2 |
+| --- | --- | --- | --- |
+| Soroban Txn per ledger | 1 | 100 | 125 |
```

Comment:
@leighmcculloch, @dmkozh, or @MonsieurNicolas 

Would you mind providing insight here? I've done my best to translate what I understand from the following proposed config update: [link](https://stellar.expert/explorer/public/staged-soroban-config/vMdIBVCY7OcPuX3DJxQQg6dyszwHIgzBbBNGQuxPxOkrPqTMhlrObmws6vnhmxMBzQkZHVj8IfiJvJtiodXG9Q%3D%3D) but may have missed some things